### PR TITLE
Added GetDrugExposureEndDate function

### LINKED ISSUE #69: [FEATURE] Add New Function `GetDrugExposureEndDate`
The following code gets the end dates associated with a particular drug exposure:

```julia
function GetDrugExposureEndDate(
    drug_exposure_ids;
    tab = drug_exposure
)

    sql =
        From(tab) |>
        Where(Fun.in(Get.drug_exposure_id, drug_exposure_ids...)) |>
        Select(Get.drug_exposure_id, Get.drug_exposure_end_date) |>
        q -> render(q, dialect=dialect)

    return String(sql)

end

function GetDrugExposureEndDate(
    drug_exposure_ids,
    conn;
    tab = drug_exposure 
)

    df = DBInterface.execute(conn, GetDrugExposureEndDate(drug_exposure_ids; tab=tab)) |> DataFrame

    return df
end
```

I just haven't had the time yet to add this code into CohortCreator as a new feature. Would be nice to add it in. The implementation is exactly the same as PR #59 and requires a bit more documentation, tests, and a dispatch for DataFrames. 

A great issue for beginners and to get started!

### PR DESCRIPTION
**Pull request by Jay-sanjay**
This PR is related to #69 

## PR CheckList

- [x]  Added GetDrugExposureEndDate function
- [x]  Added doc-strings and test-suite for it
- [x]  Added a multiple function dispatch

closes #69

### CODE REVIEW

File: src/getters.jl [hunk 1]
```diff
@@ -1885,4 +1885,90 @@ function GetDatabaseCohorts(
     
 end
 
-export GetDatabasePersonIDs, GetPatientState, GetPatientGender, GetPatientRace, GetPatientAgeGroup, GetPatientVisits, GetMostRecentConditions, GetMostRecentVisit, GetVisitCondition, GetPatientEthnicity, GetDatabaseYearRange, GetVisitPlaceOfService, GetVisitConcept, GetVisitDate, GetDrugExposures, GetDrugConceptIDs, GetDrugAmounts, GetVisitProcedure, GetDatabaseCohorts, GetCohortSubjects, GetCohortSubjectStartDate, GetCohortSubjectEndDate, GetDrugExposureIDs
+"""
+function GetDrugExposureEndDate(drug_exposure_ids; tab = drug_exposure)
```
Comment (TheCedarPrince) [inline src/getters.jl:1889]: Could you add the `conn` argument into this docstring?

File: src/getters.jl [hunk 2]
```diff
@@ -1885,4 +1885,90 @@ function GetDatabaseCohorts(
...
+
+Given a list of drug_exposure IDs, find their corresponding drug_exposure_end_date ID.
```
Comment (TheCedarPrince) [inline src/getters.jl:1891]: I'd say instead:

"Given a list of drug_exposure IDs, find their exposure end dates."

File: src/getters.jl [hunk 3]
```diff
@@ -1885,4 +1885,90 @@ function GetDatabaseCohorts(
...
+    df = DBInterface.execute(conn, GetDrugExposureEndDate(drug_exposure_ids; tab=tab)) |> DataFrame
+
+    return df
+end
+
+"""
+function GetDrugExposureEndDate(df:DataFrame, conn; tab = drug_exposure)
+
+Given a `DataFrame` with a `:drug_exposure_id` column and `:drug_exposure_end_date` column, return the `DataFrame` with an associated 
+`:drug_exposure_end_date` corresponding to a given `drug_exposure_id` and `drug_exposure_end_date` in the `DataFrame`
```
Comment (TheCedarPrince) [inline src/getters.jl:1924]: I'd rewrite this as:

Given a `DataFrame` with a `:drug_exposure_id` column, return the `DataFrame` with an associated 
`:drug_exposure_end_date` corresponding to a given `drug_exposure_id` in the `DataFrame`.

File: src/getters.jl [hunk 4]
```diff
@@ -1885,4 +1885,90 @@ function GetDatabaseCohorts(
...
+- `drug_exposure_ids` - list of `drug_exposure_id`'s; each ID must be of subtype `Float64`
+
+
+# Keyword Arguments:
+
+- `tab` - the `SQLTable` representing the Drug Exposure table; default `drug_exposure`
+
+# Returns
+
+- `df::DataFrame` - a two column `DataFrame` comprised of columns: `:drug_exposure_id` and `:drug_exposure_end_date`
```
Comment (TheCedarPrince) [inline src/getters.jl:1957]: This will return the SQL statement -- could you update this?

**Review by TheCedarPrince — CHANGES_REQUESTED** [review]
Just a few changes mostly to documentation. Also, could you add a reference to this function in the API page of the documentation?

Thanks and great work on this -- you made this super straightforward and nice to review!

**Review by TheCedarPrince — APPROVED** [review]
LGTM! Feel free to merge.

Question: Do you still want to explore another PR that improves code coverage separately? Or did you want to try exploring that further in this PR?

**Comment by Jay-sanjay** [comment]
@TheCedarPrince  yes that might be a good idea,I can file an issue for it. Because if in case I won't able to add that for all function, then also the issue would be present for the future work.

**Pull request merged**